### PR TITLE
Created Cart page

### PR DESCRIPTION
**Pull request by rahulcs754**
- added cart page
image

Please check

### CODE REVIEW

**Review by kjais1720 — COMMENTED** [review]
Looks good overall, but one suggestion, in a single PR please push changes only for the particular page that you have created the branch for

**Review by kjais1720 — APPROVED** [review]
looks nice and clean with good usage of utility classes 👍

File: assets/css/custom.css [hunk 1]
```diff
@@ -35,7 +84,12 @@
     border: 1px solid var(--light-black);
 }
 
-.space-between{
+.space-between,
+.mycart-amout-desc{
```
Comment (tanishq20) [inline assets/css/custom.css]: typo: `amount`

**Review by tanishq20 — COMMENTED** [review]
Looks good. Suggested very small change.

File: assets/css/custom.css (hunk 1 above)
Comment (rahulcs754) [reply assets/css/custom.css]: correct amount

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**